src/AssetsPage/AssetsTable/AssetsTable.test.jsx

```diff
@@ -74,7 +81,31 @@ describe('<AssetsTable />', () => {
       expect(wrapper.find('tr')).toHaveLength(defaultProps.assetsList.length + 1);
     });
     it('correct number of trashcan buttons', () => {
-      expect(wrapper.find('button').filterWhere(button => button.hasClass('fa-trash'))).toHaveLength(defaultProps.assetsList.length);
+      expect(wrapper.find('tr button').filterWhere(button => button.hasClass('fa-trash'))).toHaveLength(defaultProps.assetsList.length);
```

Comment:
I thought it might be good to specify that this trash button is in a table row and not just floating around the page.